spec/services/idv/profile_maker_spec.rb

```diff
@@ -92,12 +94,32 @@
       end
     end
 
+    context 'with in_person_verification_needed' do
+      it 'deactivates a profile for in person verification' do
+        profile = subject.save_profile(
+          fraud_pending_reason: nil,
+          gpo_verification_needed: false,
+          deactivation_reason: nil,
+          in_person_verification_needed: true,
+        )
+
+        expect(profile.activated_at).to be_nil
+        expect(profile.active).to eq(false)
+        expect(profile.deactivation_reason).to eq('in_person_verification_pending')
```

Comment:
Should there be an expectation for profile.in_person_verification_pending_at on this spec and the others in this file?

Reply:
also `ProfileSpec`